pkg/cmd/policy_validate.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2016-2020, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/pulumi/pulumi/pkg/backend"
+	"github.com/pulumi/pulumi/sdk/go/common/util/cmdutil"
+	"github.com/spf13/cobra"
+)
+
+func newPolicyValidateCmd() *cobra.Command {
+	var argConfig string
+
+	var cmd = &cobra.Command{
+		Use:   "validate <org-name>/<policy-pack-name> <version>",
+		Args:  cmdutil.ExactArgs(2),
+		Short: "[PREVIEW] Validate Policy Pack configuration against config schema",
+		Long:  "Validate Policy Pack configuration against the specified version of its configuration schema.",
+		Run: cmdutil.RunFunc(func(cmd *cobra.Command, cliArgs []string) error {
+			// Obtain current PolicyPack, tied to the Pulumi service backend.
+			policyPack, err := requirePolicyPack(cliArgs[0])
+			if err != nil {
+				return err
+			}
+
+			// Get version from cmd argument
+			version := &cliArgs[1]
+
+			// Load the configuration from the user-specified JSON file into config object.
+			var config map[string]*json.RawMessage
+			if argConfig != "" {
```

Comment:
If argConfig is empty, this will just validate an empty config? Seems like a waste, but maybe fine since the config could require values and then an empty config would be invalid... wondering if we should output some warning thought like BTW you didnt give us config? Seems like it could be a very easy mistake to make from a user perspective.

Reply:
Good point! yea, for whatever reason idk why I was assuming this might be covered by marking the flag as required, but now coming to think of it, it probably would allow to run with not specifying anything and be a bad user experience... will address. thanks.

Reply:
ok. just tested it. the way it is written, it actually will not let you proceed without specifying a file and print a message telling the user they need to specify a config file, which I believe is the behavior we want. I did however go and add an integration test though to verify behavior with file that is empty and file that has `{}`.